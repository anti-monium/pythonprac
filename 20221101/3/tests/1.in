a = Grange(2, 3, 100000)
b = Grange(1, 4, 100500)
c = Grange(3, 5, 123456789)
print(len(a) + len(b) + len(c))
if bool(a[1:1]):
	print('bad boy')
else:
	print('good boy')
print(c, c[0], c[6], c[10])
print(*list(b[4:10005]))
print(a[6:19999:2], *list(a[6:19999:2]))
