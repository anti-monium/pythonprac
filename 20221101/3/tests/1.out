30
good boy
grange(3, 5, 123456789) 3 46875 29296875
4 16 64 256 1024 4096
grange(6, 9, 19999) 6 54 486 4374
